src/highlight/HighlightAnnotations.tsx

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
Consider importing this as `import uuid from 'uuid'` and using `uuid.v4()` below?